skiko/src/awtMain/kotlin/org/jetbrains/skiko/redrawer/WindowsOpenGLRedrawer.kt

```diff
@@ -73,6 +74,8 @@ internal class WindowsOpenGLRedrawer(
     private fun swapBuffers() = swapBuffers(device)
 
     companion object {
+        @OptIn(ExperimentalCoroutinesApi::class)
+        private val dispatcherToBlockOn = Dispatchers.IO.limitedParallelism(64)
```

Comment:
We can't use experiemental API in skiko, as it isn't backward compatible.

This configuration will throw an exception in runtime:

App -> Kotlin 1.20 -> non-compatible-time API
App -> Compose -> Skiko -> Kotlin 1.8 -> non-compatible-time API

Reply:
Why we use limitedParallelism? the default IO has the limit 64, if I am not mistaken.

Reply:
> Isn't it safe to use internally, if we wont expose it in public api?

It isn't safe, as an application can use different Kotlin. When it uses newer Kotlin, Skiko automatically uses it too. And if there are incompatible changes (for example, limitedParallelism is renamed), there will be a crash in skiko.

Reply:
We can add an argument to `SkiaLayer(dispatcherToBlockOn = Executors.newFixedThreadPool(64).asCoroutineDispatcher())` (or maybe even `newCachedThreadPool`) and pass it down. In Fleet we can pass `limitedParallelism`.

Reply:
After thinking more, it seems `newCachedThreadPool` is the better option for vsync for all clients (Compose/Fleet)? It doesn't reuse threads from the common coroutines pool though, but not sure if it adds significant overhead.

Reply:
Maybe the ability to reuse threads from the common coroutines pool is not a big problem. I think in most cases we switch from UI thread to this executor and UI thread is not in the common pool too.